src/test/java/org/minions/devfund/marylin/SquarelotronTest.java

```diff
@@ -0,0 +1,634 @@
+package org.minions.devfund.marylin;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * SquarelotronTest class.
+ */
+public class SquarelotronTest {
+    private static final Logger LOGGER = Logger.getLogger(WhackAMole.class.getName());
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+    private static final int TWO = 2;
+    private static final int THREE = 3;
+    private static final int FOUR = 4;
+    private static final int FIVE = 5;
+    private static final int NEGATIVEONE = -1;
+    private static final int NEGATIVETWO = -2;
+    private static final int NEGATIVETHREE = -3;
+    private static final int NEGATIVEFOUR = -4;
+    private static final int NEGATIVEFIVE = -5;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "NEGATIVEFIVE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)